cmake/SetupCompiler.cmake

```diff
@@ -180,5 +196,4 @@ function(xad_add_executable name)
         RELWITHDEBINFO_POSTFIX ""
         MINSIZEREL_POSTFIX ""
     )
-endfunction()
-
+endfunction()
```

Comment:
We're in general trying to keep the diff small and only for the relevant/changed parts. This avoids potential conflicts for other developers. Removing this newline at the end of the file is not necessary.